supervision/annotators/core.py

```diff
@@ -847,6 +848,7 @@ def __init__(
         self.radius: int = radius
         self.position: Position = position
         self.color_lookup: ColorLookup = color_lookup
+        self.outer_thickness = outer_thickness
```

Comment:
Let's keep the naming the same as much as possible. `outline_thickness` sounds correct here too.